src/components/changePassword/ChangePasswordPage.tsx

```diff
@@ -1,22 +1,42 @@
-import React, { useState } from 'react'
+import React from 'react'
 import { useTranslation } from 'react-i18next'
 import { Container, Grid, TextField, Button } from '@material-ui/core'
+import { useFormik } from 'formik'
+import * as yup from 'yup'
 
 import Layout from 'components/layout/Layout'
 
 export default function ChangePasswordPage() {
   const { t } = useTranslation()
-  const [password, setPassword] = useState('')
-  const [confirmPassword, setConfirmPassword] = useState('')
 
-  function handleSubmit(event: React.FormEvent) {
-    event.preventDefault()
-  }
+  const ChangePasswordSchema = yup.object().shape({
+    password: yup
+      .string()
+      .min(6, t('auth:validation.password-min', { count: 6 }))
```

Comment:
It's gonna be better to handle the translation at the last step right before displaying the error.

Otherwise we risk to receive mixed translations as the messages are memoized.

We can handle similar validations in a central location as it's shown at https://github.com/jquense/yup#using-a-custom-locale-dictionary

This will also help us decompose the validation schema definition in a separate file, away from the actual form component implementation.

Not super important at this stage but in future can be confusing.

https://user-images.githubusercontent.com/893608/107115221-c0329400-6873-11eb-9b00-d7163d3ca56e.mp4